org_fedora_oscap/content_handling.py

```diff
@@ -50,6 +51,47 @@ class ContentCheckError(ContentHandlingError):
 
     pass
 
+class ParseHTMLContent(HTMLParser):
+    """Parser class for HTML tags within content"""
+
+    def __init__(self):
+        HTMLParser.__init__(self)
+        self.content = ""
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "html:ul":
+            self.content = self.content + "\n"
+        elif tag == "html:li":
+            self.content = self.content + "\n"
+        elif tag == "html:br":
+            self.content = self.content + "\n"
```

Comment:
I am wondering how the parser works, it adds two newlines per li and ul and only one newline for br. Can you elaborate on these choices?